### DISCUSSION TITLE
Tentative Questline for Temple characters

### DISCUSSION THREAD

**Discussion by Maligerent** [globemallow-mw/ossuaries-of-the-ghostfence#12] [Ideas]
1. Similar to Seven Graces, the character is prompted at Ghostgate to visit all three ossuaries to receive their blessings. (Choice of overland travel like usual, or through the catacombs.) Once close to the catacomb wall in proximity to Kogoruhn's back entrance, there will be evidence of a partially collapsed wall and 3D sound of digging - the Sixth House at this stage has not entered the catacombs yet, but is looking for a way in. the ossuary of Vehk will have a small contingent of Buoyant Armigers and little else - optional side quest there of taking a letter requesting reinforcements and supplies back to Ghostgate.
2. Once the Pilgrimage of the Dead is complete, the player is directed to the leader of the Order of Ghosts. The leader explains that they are in need of more donations. Player does a check-in with families in Ald-ruhn, Vivec, Gnisis, and Maar Gan as well as elderly NPCs to see if they are willing to donate remains to reinforce the Ghostgate. 
3. The Order of Ghosts needs the player to investigate an influx of non-Dunmer remains (guar and other animals). First indication that the Temple has actively been purchasing remains from graverobbers to bolster the Ghostfence, confirmed when player visits the graverobbers' location and clears the enemies/finds a letter and ledger describing the transactions.
4. Order of Ghosts confirms the transactions and admits that holding back Dagoth Ur is starting to become increasingly more difficult. To give the player a scope of their desperation, they are instructed to visit the ancestral tombs connected to the catacombs and help the Order of Ghosts reps there, showing that they have resorted to simply taking remains that have not been donated.
5. returning to the leader, player learns that they have received reports from both Ayem and Seht ossuaries that the 6th house has broken into the catacombs. Player is instructed to help Seht first, as they are smaller. Player learns they are absolutely not defenseless-- they have operational clockwork defenses that are doing a great job of keeping the monsters out. The Acolytes tell the player to assist Ayem instead as the Ordinators have resorted to barricading the catacomb tunnels. Once the player helps get rid of the corprus monsters and other low-level 6th house creatures they learn that not only has the Ossuary of Vehk not sent runners, but the people who sent supplies their way have not returned either. Player is instructed to head through the catacombs to find out what happened.
6. The catacombs from Ayem to Vehk are overrun with 6th house agents - this leg of the catacombs now has the 6th house look to it. (temporary replacement of interior cells) Player fights through and can choose to repair the wall themselves to prevent further reinforcements. (Or block it off by collapsing it?) The Ossuary itself is overrun as well, the suppliers and Armigers now dead. Here is where the player will possibly meet Dagoth Aranen if we decide to have a boss battle, as well as numerous other 6th house monsters.
7. Player can now help rebuild the Vehk Ossuary and pay for its upkeep, possibly interop with Danae's necromancy mod to donate Dunmer skulls to the Order of Ghosts. Once Dagoth Ur is defeated, pilgrims will start attending Vehk again, giving the player a certain amount of gold monthly.

**Comment by globemallow-mw** [comment]
> Similar to Seven Graces, the character is prompted at Ghostgate to visit all three ossuaries to receive their blessings.

What should the blessings be? Making a note to discuss as a team.

> Once close to the catacomb wall in proximity to Kogoruhn's back entrance, there will be evidence of a partially collapsed wall and 3D sound of digging

I imagine this "digging" to be a kind of incessant scraping at the wall, akin to psychological warfare.

> Player does a check-in with families in Ald-ruhn, Vivec, Gnisis, and Maar Gan as well as elderly NPCs to see if they are willing to donate remains to reinforce the Ghostgate.

"Ancestors and the Dunmer" (see [Publisher's Note](https://en.uesp.net/wiki/Morrowind:Ancestors_and_the_Dunmer) at the bottom) suggests that the "donation" isn't really voluntary anymore. Maybe the player gets a list of the recently deceased from the Order and has to go collect remains. Some families are more willing to give them up than others. Another possible side quest: someone reports an "illegal" clan ghost fence and the player has to investigate.

> The Order of Ghosts needs the player to investigate an influx of non-Dunmer remains (guar and other animals). First indication that the Temple has actively been purchasing remains from graverobbers to bolster the Ghostfence, confirmed when player visits the graverobbers' location and clears the enemies/finds a letter and ledger describing the transactions.

Maybe the Temple is offering amnesty to graverobbers in exchange for remains? Why are there enemies in the graverobbers' location? Alternatively, the player is sent by the Temple to clear out the graverobbers, and the collusion with the Temple comes as a surprise at the end! The Temple was just using the player to try and cover its tracks. The influx of animal remains can be a separate side-quest?

> they are instructed to visit the ancestral tombs connected to the catacombs and help the Order of Ghosts reps there, showing that they have resorted to simply taking remains that have not been donated.

Fetch quest? "Bring me 10 bonemeal, please." :)

> they have operational clockwork defenses that are doing a great job of keeping the monsters out. 

I wish we could make the Dwemer ballista functional!

> Player fights through and can choose to repair the wall themselves to prevent further reinforcements. 

Maybe the player can make some choices earlier in the questline (like delivering the letter requesting reinforcements) that impacts the Ossuary's readiness when the Sixth House attacks. Repairing the wall could be one of them.

> Player can now help rebuild the Vehk Ossuary and pay for its upkeep.

This feels like a bit of feature creep. Definitely worth thinking about if we have time, though!

> The catacombs from Ayem to Vehk are overrun with 6th house agents - this leg of the catacombs now has the 6th house look to it. (temporary replacement of interior cells)

I like this idea, but it does create double the work (creating two different sets of interiors.)

**Reply by dev-chenxing** [reply]
Redundant post (new post in issue Quest 1)

> What should the blessings be? Making a note to discuss as a team.

- Shrine of the Body (Vehk): one soul went to the grave with the "body". The blessing is Sanctuary 10 pts for 1440 sec. The prayer is "We brought nothing into this world, neither may we carry anything out of this world. Lord Vivec giveth, and Lord Vivec taketh away."
- Shrine of the Reborn (Ayem): one soul went through the judgment and was eventually reborn. The blessing is Restore Health 500 pts for 1 sec, Restore Magicka 500 pts for 1 sec, Restore Fatigue 500 pts for 1 sec. The prayer is "".
- Shrine of the Altar (Seht): one soul remained in or near the ancestral altar. The blessing is Detect Animal 50 pts for 1440 sec

I suggest all of the shrine do not need donation in order to ask for a blessing due to their relatively remote location.

**Comment by globemallow-mw** [comment]
The "Order of Ghosts" comes across as a kind of rogue Temple faction, with a brutal "the end justifies the means" worldview. They don't see a problem with taking more extreme measures to combat the Blight, like working with graverobbers or stealing remains. Maybe we should we give the player a choice between leaning into this mindset, or reforming the Order? What would that look like?

I also think in the progression of the questline, the ruthless nature of the Order should be slowly revealed to the player.

**Comment by dev-chenxing** [comment]
> 2. Once the Pilgrimage of the Dead is complete, the player is directed to the leader of the Order of Ghosts. 

I take the second quest assignment and it might include the placement of an important NPC, the leader of the Order of the Ghosts, whom I am gonna to randomly name Adansa Kren (AK) for now until someone comes up with a better one.

**Reply by Kalinter** [reply]
Since we decided to make the Order the primary faction of the mod I decided to just have you meet the leader in the first quest and you join through them. You see Quest 1 now for details. I can change though if it doesn't fit later, and have you be recruited by someone else. That name sounds fine and unique. I just used a placeholder for their name in the first quest. For such an important character though we will want to decide a bit about their background together. I assume Dunmer of course. Male or Female? What Class?

**Reply by dev-chenxing** [reply]
> we will want to decide a bit about their background together. I assume Dunmer of course. Male or Female? What Class?

I agree that they should be the quest giver of the first quest. I suggest Female and a new class that skilled in the magic schools that the faction members is required, light armor and long blade as their background. Also they probably shouldn't wear a hood. ![image](https://user-images.githubusercontent.com/109032964/193463985-45302138-db98-495b-905d-34091d5b4491.png)
